src/test/scala/kamon/newrelic/metrics/NewRelicDistributionMetricsSpec.scala

```diff
@@ -25,7 +25,7 @@ class NewRelicDistributionMetricsSpec extends WordSpec with Matchers {
         .put("dimension", "information")
         .put("sourceMetricType", "mountain")
       val summary = new Summary("trev.summary", 44, 101.0, 13.0, 17.0, TestMetricHelper.start, TestMetricHelper.end, summaryAttributes)
-      val gaugeAttributes = summaryAttributes.copy().put("percentile.countAtRank", 816L).put("percentile", 19.0d)
+      val gaugeAttributes = summaryAttributes.copy().put("percentile.countAtRank", 816L).put("percentile", 90.0d)
```

Comment:
I hate to say it, but we probably want to have an unwanted percentile in the source data as well, so that we can verify that it doesn't make it into the output.

Reply:
Or maybe you already accounted for that and I'm just not seeing it...

Reply:
no, that's a good call.  will add